packages/grafana-toolkit/src/config/webpack/loaders.ts

```diff
@@ -33,39 +31,6 @@ export const getStylesheetEntries = (root: string = process.cwd()) => {
   return entries;
 };
 
-export const hasThemeStylesheets = (root: string = process.cwd()) => {
```

Comment:
This function is not longer used, originally it was trying to figure if the developer forgot to add a light/dark theme in the plugin but this is "too smart" for the builder to do.